DataFormats/TauReco/src/classes_def_3.xml

```diff
@@ -99,7 +99,11 @@
   <class name="std::pair<reco::PFTauRef, reco::PFTauDecayMode>"/>
   <class name="std::vector<std::pair<reco::PFTauRef, reco::PFTauDecayMode> >" />
 
-  <class name="reco::PFTauTransverseImpactParameter"/>
+  <class name="reco::PFTauTransverseImpactParameter" ClassVersion="11">
+   <version ClassVersion="11" checksum="2154806108"/>
+   <version ClassVersion="10" checksum="572950680"/>
+  </class>
+
```

Comment:
I tested the following modification in 76X and it compiles OK.

git diff --color  CMSSW_7_5_X_2015-06-26-2300 DataFormats/TauReco/src/classes_def_3.xml
```diff
   <class name="std::pair<reco::PFTauRef, reco::PFTauDecayMode>"/>
   <class name="std::vector<std::pair<reco::PFTauRef, reco::PFTauDecayMode> >" />
 
-  <class name="reco::PFTauTransverseImpactParameter"/>
+  <class name="reco::PFTauTransverseImpactParameter" ClassVersion="2">
+   <version ClassVersion="2" checksum="2154806108"/>
+  </class>
+
```